crystal-map-connector/src/main/kotlin/ch/ergon/dope/CrystalMapAdapter.kt

```diff
@@ -26,25 +26,25 @@ fun CMType.asField(reference: String = path): Field<out ValidType> = Field(
     reference,
 )
 
-@JvmName("asNumberField")
-fun CMField<out Number>.asField(reference: String = path): Field<NumberType> = Field(name, reference)
+@JvmName("toDopeNumberField")
+fun CMField<out Number>.toDopeField(reference: String = path): Field<NumberType> = Field(name, reference)
 
-@JvmName("asStringField")
-fun CMField<String>.asField(reference: String = path): Field<StringType> = Field(name, reference)
+@JvmName("toDopeStringField")
+fun CMField<String>.toDopeField(reference: String = path): Field<StringType> = Field(name, reference)
 
-@JvmName("asBooleanField")
-fun CMField<Boolean>.asField(reference: String = path): Field<BooleanType> = Field(name, reference)
+@JvmName("toDopeBooleanField")
+fun CMField<Boolean>.toDopeField(reference: String = path): Field<BooleanType> = Field(name, reference)
 
-@JvmName("asNumberArrayField")
-fun CMList<out Number>.asArrayField(): Field<ArrayType<NumberType>> = Field(name, path)
+@JvmName("toDopeNumberArrayField")
+fun CMList<out Number>.toDopeArrayField(): Field<ArrayType<NumberType>> = Field(name, path)
```

Comment:
Can't we just call this extension function `toDopeField()` as well, for even more uniformity? Does the user care if it is an array, or should they have to?